README.md

```diff
@@ -27,8 +27,8 @@ Hover over a table name to see the table's definition.
 
 ### Find all references
 
-Click on the table name in your schema definition to see all places it is used
-(its references).
+Command/control-click on a field name in your schema definition to see all places
```

Comment:
The example shows a field name, so I assume it must be “field name” here? Or are both supported and the paragraph should say something like “on a table or field name in your schema definition”?